### ISSUE TITLE
This is an official Google plugin, and it's abandoned

### ISSUE THREAD

**Issue by jpike88** [google/cordova-plugin-browsertab#41]
Says it all. Now I can't use it with cordova-android 10, because it still relies on cordova-compat, which was deprecated 4 YEARS AGO.

And forking it to remove the dependency doesn't sort it either, because this plugin's format is so old that Cordova can't pull directly from the GitHub repo.

Good luck to anyone who doesn't enjoy wasting their own time.

**Comment by Mister-CK** [comment]
I fully agree that it is negligent of Google to just abandon their plugins (although I am not sure if this was ever an official google plugin or just something one of their employees made). However, we currently use this plugin directly and we don't have codova-compat in our codebase so it should be usable without it. 
In the past we have also used a fork of this plugin without issues. So I don't understand why you are running into issues there.
I do think your problems are solvable within a couple of days.
